src/components/ClusterManager.js

```diff
@@ -330,6 +332,54 @@ export class NewClusterModal extends PureComponent {
   }
 }
 
+const ClusterErrorModal = ({ cluster, onDismiss }) => {
+  const [error, setError] = useState()
+  const [userscriptError, setUserscriptError] = useState(false)
+  const [loadingClusterDetails, setLoadingClusterDetails] = useState(false)
+
+  const loadClusterError = _.flow(
+    withErrorReporting('Error loading cluster details'),
+    Utils.withBusyState(setLoadingClusterDetails)
+  )(async () => {
+    const { errors: clusterErrors } = await Ajax().Jupyter.cluster(cluster.googleProject, cluster.clusterName).details()
+    if (_.some(({ errorMessage }) => errorMessage.includes('Userscript failed'), clusterErrors)) {
+      setError(await Ajax().Buckets.getObjectPreview(cluster.stagingBucket, `userscript_output.txt`, cluster.googleProject, true).then(res => res.text()))
+      setUserscriptError(true)
+    } else {
+      setError(clusterErrors[0].errorMessage)
+    }
+  })
+
+  Utils.useOnMount(() => { loadClusterError() })
+
+  return h(Modal, {
+    title: userscriptError ? 'Cluster Creation Failed due to Userscript Error' : 'Cluster Creation Failed',
+    onDismiss
+  }, [
+    div({ style: { whiteSpace: 'pre-wrap', overflowWrap: 'break-word', overflowY: 'auto', maxHeight: 500, background: colors.gray[6] } }, [error]),
```

Comment:
This will need to be changed to use the new color palette functions. Be sure to rebase onto the latest dev and test before merging.